driver/terraform.go

```diff
@@ -547,6 +541,26 @@ func (tf *Terraform) initTaskTemplate() error {
 	return nil
 }
 
+func (tf *Terraform) setNotifier(tmpl templates.Template, serviceCount int) {
```

Comment:
I like this new separate method for setting the notifier since we have so many cases now!